examples/drivenet/drivenet_close_loop.ipynb

```diff
@@ -0,0 +1,329 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DriveNet Closed-Loop Evaluation\n",
+    "\n",
+    "**Note: this notebook assumes you've already run the [training notebook](./drivenet_train.ipynb) and stored your model successfully.**\n",
+    "\n",
+    "## What is a closed-loop evaluation?\n",
+    "In closed-loop evaluation DriveNet is in **full control of the AV**. At each time step, we predict the future trajectory and then move the AV int he first of the DriveNet's predictions. \n",
+    "\n",
+    "At Lyft, we refer to this process with the terms **forward-simulate** or **unroll**.\n",
+    "\n",
+    "![closed-loop](../../images/drivenet/closed-loop.svg)\n",
+    "\n",
+    "This evalaution is crucial to assess the real performance of our DriveNet. **Ideally, we would test the model on the road in the real world**. However, this is clearly very expensive and scales poorly. Forward-simulation is an attempt to evaluate the system in a setting which is closest as possible to a real road test on the same route.\n",
+    "\n",
+    "\n",
+    "## What is a good closed-loop metric?\n",
+    "In this setting **metrics are particularly challenging**. In fact, we would like to penalise some of the simulation drift (e.g. going off road or in the opposite lane) while at the same time allow others (e.g. different speed profiles). As an example, using the same metrics we employed during the open loop evaluation (e.g. ADE) would penalise DriveNet Unconditionally.\n",
+    "\n",
+    "At Lyft L5, we use a large set of different metrics to capture dangerous manoeuvres and behaviours. \n",
+    "\n",
+    "For the sake of simplicity, in this notebook we will be using a very simple proxy to detect if our model is driving in a sensible way, composed of two different metrics:\n",
+    "\n",
+    "### Collisions\n",
+    "Our AV should avoid collisions with other agents. However, not all collisions are created equally. This is because while our AV is fully controlled by Drivenet, other agents are not (a setting we call **log replay**).\n",
```

Comment:
```suggestion
    "Our AV should avoid collisions with other agents. However, not all collisions are created equally. This is because while our AV is fully controlled by DriveNet, other agents are not (a setting we call **log replay**).\n",
```